config.json.sample

```diff
@@ -0,0 +1,5 @@
+{
+  "debug": false,
+  "featureManagerIsFullZoneProvisioningEnabled": true,
+  "locale": "en"
+}
```

Comment:
Yes, that makes more sense. We've changed the power from the json file to the json response from `/config` route. Meaning "Building Your Own Backend" should have this piece of knowledge.